.github/workflows/docs.yml

```diff
@@ -0,0 +1,23 @@
+name: Docs
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+    - name: Build docs
+      run: |
+        pip install tox
```

Comment:
Probably we should use `pip install tox~=VERSION` here, or `pip install --upgrade tox`.

Reply:
As I understand it we get a [fresh virtual environment](https://help.github.com/en/articles/virtual-environments-for-github-actions) for each run, but I suppose there's no reason _not_ to use `--upgrade`.

I think it's fine to use current `tox` here since the dependency won't affect users, and there's some benefit to staying up to date.